kernel/src/main/scala-2.12/cats/kernel/ScalaVersionSpecificLazyListCompat.scala

```diff
@@ -0,0 +1,9 @@
+package cats
+package kernel
+
+private[kernel] object ScalaVersionSpecificLazyListCompat extends LazyListCompatBase {
```

Comment:
I wonder, can we simply define a type alias Stream => LazyList for Scala 2.12,
while keeping a usual LazyList for Scala 2.13+ ?
```
type LazyList[a] => Stream[a]
val LazyList = Stream
```
or something like that?

Reply:
We could probably do that, but we'd be implying that all methods used would be identical for `Stream` and `LazyList`. I think they are right now...and we could add exceptions if needed in the future...

So...yes? I'll try it out.